net-im/synapse/files/synapse.initd

```diff
@@ -0,0 +1,39 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+description="Reference implementation of Matrix homeserver"
+description_reload="Reload configuration and reopen log files."
+
+su=/bin/su
```

Comment:
what
Why don't you use start-stop-daemon like all the other openrc scripts?

Reply:
Because this crap uses apachectl-like binary to start separate process and there is no viable way to run it by hand because it does config checks and stuff. This is piece of crap and it even doesn't produce .pid file in predetermined place.

Reply:
I am really open to suggestions how to make it look better